src/ui/menu/Menu.ts

```diff
@@ -273,11 +296,40 @@ export class Menu {
     this.menuClose.emit();
   }
 
+  public set Title(text: string) {
+    this.title = text;
+    this.titleResText.caption = text;
+  }
+
+  public get Title() {
+    return this.title;
+  }
+
   public set Subtitle(text: string) {
     this.subtitle = text;
     this.subtitleResText.caption = text;
   }
 
+  public get Subtitle() {
```

Comment:
> You seem to change to pascal case for the sets and gets?

subtitle set method was already is pascal case so I just continued with it + there is private subtitle declared at the top. It would result in duplicate identifier if i changed Subtitle to subtitle

Reply:
> You seem to change to pascal case for the sets and gets?

Pascal case is the format set for getters and setters using the `tslint-config-airbnb` that this project uses.

Therefore, his code formatting looks okay.